packages/h/hexo-theme-keep.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "hexo-theme-keep",
+  "description": "A simple and elegant theme for Hexo.",
+  "keywords": [
+    "hexo",
+    "theme",
+    "simple",
+    "elegant",
+    "powerful"
+  ],
+  "license": "AGPL-3.0",
+  "homepage": "https://github.com/XPoet/hexo-theme-keep#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/XPoet/hexo-theme-keep.git"
+  },
+  "authors": [
+    {
+      "name": "XPoet",
+      "email": "i@xpoet.cn",
+      "url": "https://xpoet.cn"
+    }
+  ],
+  "autoupdate": {
+    "source": "npm",
+    "target": "hexo-theme-keep",
+    "fileMap": [
+      {
+        "basePath": "source",
+        "files": [
+          "js/**/*",
+          "font/**/*",
+          "images/**/*"
```

Comment:
Please avoid using unrestricted globstars, as they can lead to including resources on the CDN that aren't listed. Please restrict the extensions you are matching, and ideally the directories as well.

Reply:
I have modified it, please help me merge it, thank you very much.

[commit details](https://github.com/cdnjs/packages/pull/1679/commits/9a6dfdb3daa9f83755f3652daff01226f1fee801)